pkg/controller/controller.go

```diff
@@ -522,3 +523,39 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
+// IsOwnedBy checks if the given object has a controller ownerReference set to the given owner
+func IsOwnedBy(obj metav1.Object, owner metav1.Object) bool {
+	ref := GetControllerOf(obj)
```

Comment:
@deads2k can you clarify please? do you suggest to rename the method to `IsControlledBy` or something like that?
In Kubernetes core owner and controllerRef are used interchangeably, for example https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L409
```go
	// Find the owning replica set
	...
	controllerRef := controller.GetControllerOf(pod)
	if controllerRef == nil {
		// No controller owns this Pod.
		return nil
	}
	if controllerRef.Kind != extensions.SchemeGroupVersion.WithKind("ReplicaSet").Kind {
		// Not a pod owned by a replica set.
		return nil
	}
```